test/verify_boilerplate.py

```diff
@@ -242,7 +242,7 @@ def get_regexs():
     regexs["year"] = re.compile('YEAR')
     # dates can be 2014, 2015, 2016 or 2017, company holder names can be
     # anything
-    regexs["date"] = re.compile('(2014|2015|2016|2017|2018)')
+    regexs["date"] = re.compile('(2014|2015|2016|2017|2019)')
```

Comment:
This value was erroneously changed.

```suggestion
    regexs["date"] = re.compile('(2014|2015|2016|2017|2018|2019)')
```

Reply:
fixed